credoai/lens/lens.py

```diff
@@ -331,21 +331,21 @@ def _generate_pipeline(self, pipeline):
         ValueError
             Id needs to be a string.
         """
+        if pipeline is None and self.gov:
```

Comment:
The default for `pipeline` is an empty dictionary, `{} is None` would return False always I think.